### ISSUE TITLE
Pin Clicked Event not working properly

### ISSUE THREAD

**Issue by NiklasGameDev** [Mapsui/Mapsui#880]
I'm using the MapView Implementation in Xamarin Forms.
A local MbTiles file gets loaded in the MapView and four pins are set for testing.
Two of type Pin and two of type Icon. On a single tap an integer gets incremented and displayed in a label. Now when I tap on a pin of type Pin the number gets incremented but only once per pin. When I tap on the same pin again nothing happens. When I tap on a pin of type Icon nothing happens at all.

When I remove the pin from the mapview and add a new one every tap with all important properties from e.Pin, it works with multiple taps. Still nothing on the pins with Icon.

I tried setting each newly created pin to a fixed type of pin and a fixed color of red. On pins already of type pin it worked and they became red. Pins of type Icon stayed the same which basically means the tap event isnt called at all.

Ok it had to do with a unecessary call of ShowCallout() between setting the properties of the pin.
Icon pins now increment the value and turn into red pins.

Almost everything works as expected now but I'll leave my thought process in, just in case.
I still need to create a new pin each time I tap one to be able to tap them multiple times.
Maybe I'm doing something wrong and someone can explain.

Xamarin.Forms: 4.5.0.356
Mapsui.Forms: 2.0.0-beta.39

**Comment by NiklasGameDev** [comment]
Ok, I clearly dont understand how Callouts are supposed to be used. Everything is working now without creating an new pin each time.

**Comment by charlenni** [comment]
Do you think it is possible to provide your testing source code and describe, what isn’t working?

**Comment by NiklasGameDev** [comment]
Everything is working the way I'm doing it right now. I just thought I could use callouts as an infowindow to show information about the pin but that didnt seem to work because everytime I called ShowCallout() certain things stopped working. I copied the Xamarin.Forms sample where it toggles between Show and Hide on a single tap on a pin but everytime I tapped on a pin there was no callout and I could not tap on the pin again. It might have been the callout being enabled but still invisible on top of the pin blocking the tap event. I have no idea. I'm using a Frame at the bottom of my map page now containing labels and a button to show the necessary info about the pin.
